cloud/openstack/openstackSdk/sdk.go

```diff
@@ -0,0 +1,83 @@
+// Copyright © 2017 The Kubicorn Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openstackSdk
+
+import (
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack"
+)
+
+// Sdk represents an Openstack SDK.
+type Sdk struct {
+	Compute       *gophercloud.ServiceClient
+	Network       *gophercloud.ServiceClient
+	ObjectStorage *gophercloud.ServiceClient
+}
+
+// NewSdk constructs a new Openstack SDK for the specified region.
+//
+// The following environment variable list is looked up in the user
```

Comment:
Can we please document this. Those variables should be added to the [`envar.md`](https://github.com/kris-nova/kubicorn/blob/master/docs/_documentation/envar.md) file.